sdk/core/azure_core_amqp/src/value.rs

```diff
@@ -36,6 +36,14 @@ impl AmqpList {
         Self(Vec::new())
     }
 
+    pub fn new_with_size(size: usize) -> Self {
```

Comment:
Nit: `with_capacity` is idiomatic, just like you call in the line below.

Reply:
Makes sense, fixing.